### PR TITLE
Feature/use csv lib

### PR DESCRIPTION
**Pull request by victork1996**

### CODE REVIEW

**Comment by CheziSpero** [comment]
@victork1996 Consider getting the original database using the API of the website. That way each person doesn't have to download the file before he runs the script.

File: convert_csv_records_to_database.py [hunk 1]
```diff
@@ -60,18 +76,17 @@ def convert_csv_file_to_db(csv_file_path, output_db_path):
...
+        with open(csv_file_path, "r") as csv_records_file:
+            records_reader = csv.reader(csv_records_file, dialect=csv_dialect_name)
 
-            # Parse all the records
-            while True:
-                line = csv_records_file.readline()
-                if not line:
-                    break
+            # Skip the first line, it contains headers
+            next(records_reader)
```
Comment (CheziSpero) [inline convert_csv_records_to_database.py]: Use `csv.DictReader` instead of having hardcoded indexs.

File: convert_csv_records_to_database.py [hunk 2]
```diff
@@ -85,5 +100,5 @@ def main():
     convert_csv_file_to_db(args.csv_file_path, args.output_db_path)
 
 
-if __name__ == '__main__':
+if __name__ == "__main__":
     main()
```
Comment (CheziSpero) [inline convert_csv_records_to_database.py:97]: wrap with `sys.exit(...)`

File: rarity_checker.py [hunk 3]
```diff
@@ -41,5 +41,5 @@ def main():
     check_number_rarity(args.records_db_path, args.number_to_check)
 
 
-if __name__ == '__main__':
+if __name__ == "__main__":
     main()
```
Comment (CheziSpero) [inline rarity_checker.py:45]: wrap with `sys.exit(...)`

**Review by CheziSpero — CHANGES_REQUESTED** [review]

File: convert_csv_records_to_database.py (hunk 2 above)
Comment (victork1996) [reply convert_csv_records_to_database.py:97]: `main` doesn't return anything and I'm too lazy to catch all the possible exceptions and return errors for them

File: rarity_checker.py (hunk 3 above)
Comment (victork1996) [reply rarity_checker.py:45]: `main` doesn't return anything and I'm too lazy to catch all the possible exceptions and return errors for them

File: convert_csv_records_to_database.py (hunk 2 above)
Comment (CheziSpero) [reply convert_csv_records_to_database.py:97]: That's... how `sys.exit` works.. not returning an error means success.

File: rarity_checker.py (hunk 3 above)
Comment (CheziSpero) [reply rarity_checker.py:45]: Continuing this thread in above conversation. What ever we decide there - do here.

**Pull request merged**